tests/testapps/CTCatalog/CTCatalog/Tests/CTCAlignmentTestViewController.m

```diff
@@ -82,6 +82,8 @@ - (void)drawRect:(CGRect)rect {
     CGContextStrokePath(context);
 
     CGColorSpaceRelease(colorspace);
+    CFRelease(frame);
```

Comment:
in general, rather than releasing at the end, I find it a lot harder to make mistakes about balancing the release if you do an autorelease right after the create